flink-libraries/flink-table/src/main/scala/org/apache/flink/table/runtime/aggregate/AggregateUtil.scala

```diff
@@ -729,6 +747,39 @@ object AggregateUtil {
 
     (aggFunction, accumulatorRowType, aggResultRowType)
   }
+  
+  
+     /*
+   * Function for building the processing logic for aggregating data in row bounded windows
+   *  @param namedAggregates List of calls to aggregate functions and their output field names
+    * @param inputType Input row type
+    * @param rowType Type info of row
+    * @param lowerBound the window boundary
+    * @return [[org.apache.flink.streaming.api.functions.ProcessFunction]]
+   */
+  private[flink] def createBoundedProcessingOverProcessFunction(
+    namedAggregates: Seq[CalcitePair[AggregateCall, String]],
+    inputType: RelDataType,
+    rowType: RowTypeInfo,
+    lowerBound: Int): ProcessFunction[Row, Row] = {
+
+    val (aggFields, aggregates) =
+      transformToAggregateFunctions(
+        namedAggregates.map(_.getKey),
+        inputType,
+        needRetraction = true)
+
+    val aggregationStateType: RowTypeInfo =
+      createDataSetAggregateBufferDataType(Array(), aggregates, inputType)
+
+    new BoundedProcessingOverRowProcessFunction(
+      aggregates,
+      aggFields, lowerBound,
+      inputType.getFieldCount,
+      aggregationStateType, FlinkTypeFactory.toInternalRowTypeInfo(inputType))
```

Comment:
Use the new line. 
`aggregationStateType, FlinkTypeFactory.toInternalRowTypeInfo(inputType) `->

```
aggregationStateType, 
FlinkTypeFactory.toInternalRowTypeInfo(inputType)
```